bin/publish

```diff
@@ -42,5 +42,15 @@ for image_name in "${IMAGES[@]}"; do
       docker tag "$image_name:$FULL_VERSION_TAG" "$REGISTRY/$image_name:$tag"
       docker push "$REGISTRY/$image_name:$tag"
     done
+
+    # Publish only latest to Redhat Registries
+    echo "Tagging and pushing ${REDHAT_IMAGE}"
+
+    docker tag "secretless-broker-redhat:${FULL_VERSION_TAG}" "${REDHAT_IMAGE}:${VERSION}"
+    # you can't push the same tag twice to redhat registry, so ignore errors
+    if ! docker push "${REDHAT_IMAGE}:${VERSION}"; then
+      echo 'RedHat push FAILED! (Maybe the image was pushed already?)'
+      exit 0
+    fi
```

Comment:
this section is going to get hit twice, once for image_name secretless-broker and once for image_name secretless-broker-quickstart

looking at it now I think this part needs to be broken out of the image loop and put in its own `if [ "$git_description" = "v${VERSION}" ]; then` loop

maybe there is a better way to organize it, but it doesn't seem right for it to get hit twice for no reason

Reply:
good catch - fixed!